### ISSUE TITLE
Clarity for HeatRejection fan data elements

### ISSUE THREAD

**Issue by KarenWGard** [open229/ruleset-model-description-schema#181]
The HeatRejection data group currently has two variables defining fan power: 

`fan_electric_input_power` and
`fan_nameplate_power`

however, it is unclear the difference between these two variables, and exactly what they mean.  A text search for these two variables shows that they are not currently being used by the ruleset checking tool, so we can make clarifications to their meaning without undue repercussions.

This issue came to light while studying implementation of rule 22-17, which reads "The baseline heat rejection device shall have an efficiency of 38.2 gpm/hp"

In order to calculate the heat rejection device efficiency, @mkarpman suggests that we implement the following variables for heat rejection fans in exactly the same way they are used for the Fan data group:

```
					fan_motor_nameplate_power
					fan_shaft_power
					fan_motor_efficiency
```

**Comment by JasonGlazer** [comment]
@KarenWGard can you suggest the date element names and notes language you would like to see.

**Comment by KarenWGard** [comment]
@JasonGlazer - per @mkarpman's suggestion, we use the same data element names as for the fan data group:

`fan_shaft_power - Power delivered to the fan’s shaft and does not include the mechanical drive losses. Equivalent to fan brake horsepower for inch-pound units.`

`fan_motor_efficiency - Efficiency of the fan motor at rating conditions`

`fan_motor_nameplate_power - the fan motor horsepower at rating conditions`

**Comment by JasonGlazer** [comment]
I presume these are replacing the existing fan_electric_input_power and fan_nameplate_power data elements, right?

**Comment by KarenWGard** [comment]
yes

**Comment by JasonGlazer** [comment]
Addressed in 0.0.28
